src/js/utils/css.js

```diff
@@ -94,13 +94,7 @@ function _styleValue(property, value) {
 }
 
 export function transform(element, value) {
-    style(element, {
-        transform: value,
-        webkitTransform: value,
-        msTransform: value,
-        mozTransform: value,
-        oTransform: value
-    });
+    style(element, { transform: value });
```

Comment:
Side note, if we used this extensively and want to assert that we added `will-change: transform`  to any element passed to this function at debug-player-only-runtime we could put that here behind a compile flag or in a `console.assert()`:

```
// psuedo code
if (__DEBUG__) {
    console.assert('willChange' in element.styles, 'do not forget to style elements with intend to tranform with "will-change: transform;"');
}

Reply:
We only want to use this in specific cases. `will-change: transform` is expensive to run continuously. In this case it is only applied while actively dragging the player (when we're absolutely certain it's in a changing state). The reason for this is because it is computationally expensive to run it on many elements at once, and over-applying it can actually decrease performance.

source: https://developer.mozilla.org/en-US/docs/Web/CSS/will-change